test/Geosuggest_spec.jsx

```diff
@@ -431,6 +431,26 @@ describe('Component: Geosuggest', () => {
       const suggestItems = TestUtils.scryRenderedDOMComponentsWithClass(component, 'geosuggest__item'); // eslint-disable-line max-len, one-var
       expect(suggestItems.length).to.equal(2);
     });
+
+    describe('clearing the input value', () => {
```

Comment:
Is this really clearing the input value?

Reply:
not really, it is changing the input. Changed the description of test